tools/data/mit/preparing_mit.md

```diff
@@ -10,6 +10,10 @@ First of all, you can run the following script to download the videos along with
 ```shell
 bash download_data.sh
 ```
+For better IO speed, you can also resize the original videos into smaller, more densely encoded cache by:
```

Comment:
For better decoding speed, you can resize the original videos into smaller sized, densely encoded version by: